### PR TITLE
Update a few items in the spec.

### PR DESCRIPTION
**Pull request by zhenyao**
1) clarify drawRangeELements behavior
    2) consolidate draw call extra constraints into one place
    3) Remove duplicated UniformBlock data backing constaint.

### CODE REVIEW

**Comment by zhenyao** [comment]
@kenrussell Please review
@jdashg FYI

File: specs/latest/2.0/index.html [hunk 1]
```diff
@@ -1964,28 +1927,27 @@ <h2 class="no-toc">Table of contents</h2>
...
+        Array Accesses</a>, or simply discard the arguments <em>start</em> and <em>end</em> and call <code>drawElements</code>
+        instead. In either situation, no GL errors should be generated for this cause.
       </dd>
     </dl>
 
+    <p>      
+        During calls to <code>drawElements</code>, <code>drawArrays</code>, <code>drawRangeElements</code> and their
+        instanced variants, WebGL 2 performs additional error checking beyond that specified in OpenGL ES 3.0:
+        <ul>
+          <li>If the CURRENT_PROGRAM is null, an <code>INVALID_OPERATION</code> error will be generated;</li>
```
Comment (kenrussell) [inline specs/latest/2.0/index.html]: Use &lt;code&gt; around CURRENT_PROGRAM, to match the INVALID_OPERATION error.

Comment (zhenyao) [inline specs/latest/2.0/index.html]: Done

**Comment by kenrussell** [comment]
LGTM. Excellent. Merging now.

**Pull request merged**